src/components/Cards.js

```diff
@@ -8,8 +8,8 @@ export default function Cards(props) {
   return (
     <div>
       <ul>
-        {cactusArr.map((value, index) => {
-          return <img key={index} src={value} alt={`cactus${index}`} onClick={() => props.handleClick(index)} />;
+      {props.cardDeck.map((value, index) => {
+          return <img key={value.id} src={value.name} alt={`cactus${value.id}`} onClick={() => props.handleClick(value.id)} />;
         })}
```

Comment:
Let's not do this in-line. We can pull this out into a constant function. So:
```
const generateCards = () => {
  return props.cardDeck.map((value, index) => {
    return ...
  }
}
```

Then in your return

```
return (
  <div>
    <ul>
      {generateCards()}
    </ul>
</div>);
```

By doing this, we can make the component more testable.